### PR TITLE
fix(config): update Danfoss 014G0205

### PR DESCRIPTION
**Pull request by Burriz85**
PR description here

### CODE REVIEW

File: packages/config/config/devices/0x0002/014g0205 [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+			"label": "Lifeline",
+			"maxNodes": 1,
+			"isLifeline": true
+		}
+	},
+	"paramInformation": [
+		{
+			"#": "1",
+			"label": "Valve Type",
+			"description": "Type of Valve being controlled",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0002/014g0205]:
```suggestion
```
Unnecessary description

File: packages/config/config/devices/0x0002/014g0205 [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+				{
+					"label": "NO",
+					"value": 1
+				}
+			]
+		},
+		{
+			"#": "2",
+			"label": "Heat Load Strategy",
+			"description": "Set Head Load Strategy (0 = Stacking, 1 = Spreading)",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0002/014g0205]:
```suggestion
```
dito

File: packages/config/config/devices/0x0002/014g0205 [hunk 3]
```diff
@@ -0,0 +1,84 @@
...
+				{
+					"label": "Spreading",
+					"value": 1
+				}
+			]
+		},
+		{
+			"#": "3",
+			"label": "PWM period",
+			"description": "Pulse-width modulation period",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0002/014g0205]:
```suggestion
```
dito

File: packages/config/config/devices/0x0002/014g0205 [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+				},
+				{
+					"label": "Spreading",
+					"value": 1
+				}
+			]
+		},
+		{
+			"#": "3",
+			"label": "PWM period",
```
Comment (AlCalzone) [inline packages/config/config/devices/0x0002/014g0205]:
```suggestion
			"label": "PWM Period",
```

**Review by AlCalzone — CHANGES_REQUESTED** [review]
Thanks, this is better. Just a few little changes please.

Also, you need to rename the file to `014g0205.json`, overwriting the existing one.

Just do those changes in your fork and they will show up here.

**Comment by Burriz85** [comment]
I am not able to change the name. is says: A file with the same name already exists. Please choose a different name and try again.

**Comment by AlCalzone** [comment]
You'd have to delete the other one and then rename. Or replace the contents of the existing file with the new one.
This is relatively easy if you use an offline or hosted VSCode version, see https://zwave-js.github.io/node-zwave-js/#/config-files/contributing-files?id=editor-setup

Anyways, I'll do it for you, much easier than going back and forth for such a small thing.

**Review by AlCalzone — APPROVED** [review]

**Pull request merged**